plugins/hello_world_plugin/src/hello_world_ui/src/app/hello-world/devices-data-table/services/devices-data-table.service.ts

```diff
@@ -0,0 +1,35 @@
+import {Injectable} from '@angular/core';
+import {observable, Observable} from "rxjs";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DevicesDataTableService {
+
+  constructor() {
+  }
+
+  public appendBrightData(data: Array<any>): Observable<any> {
```

Comment:
why we call this method appendBrightData? also is this service used?

Reply:
the whole service will be loaded via the UFM as a data table hook, this function will be called from the UFM side and the name of the function should be defined in the ui-config.json file, please check it https://github.com/Mellanox/ufm_sdk_3.0/pull/56/files#diff-c79067ed7fc2951a0540dcd4a0e77a0d3e7488740325d1854c01767f5d489c22R58